.github/workflows/release.yml

```diff
@@ -33,5 +33,50 @@ jobs:
           name: nugets
           path: nugets/*
           retention-days: 1
-      - name: Deploy to MyGet
-        run: dotnet nuget push nugets/Particular.EndpointThroughputCounter.${{env.MinVerVersion}}.nupkg --source https://www.myget.org/F/particular/api/v2/package --api-key ${{ secrets.MYGET_PUBLISH_API_KEY }}
+     #- name: Deploy to MyGet
+       #run: dotnet nuget push nugets/Particular.EndpointThroughputCounter.${{env.MinVerVersion}}.nupkg --source https://www.myget.org/F/particular/api/v2/package --api-key ${{ secrets.MYGET_PUBLISH_API_KEY }}       
+      - name: Build SelfContained
+        run: |
+          dotnet publish src/tool/Particular.EndpointThroughputCounter.csproj -r win-x64 `
+          -c Release `
+          -p:TargetFramework=net6.0 `
+          -p:PublishSingleFile=true `
+          -p:SelfContained=true `
+          -p:PublishReadyToRun=true `
+          -p:IncludeAllContentForSelfExtract=true `
+          -p:TargetLatestRuntimePath=true `
+          -p:DebugType=embedded `
+          -p:EnableCompressSingleFile=true `
+          -p:PackAsTool=false `
+          -o ./src/binaries
+      - name: Configure AWS credentials from Test account
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESSKEY }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRETKEY }}
+          aws-region: us-east-1
+      - name: Install AzureSignTool
+        run: dotnet tool install --global azuresigntool
+      - name: Sign binaries
+        run: |
+          AzureSignTool sign `
+          --file-digest sha256 `
+          --timestamp-rfc3161 http://timestamp.digicert.com `
+          --azure-key-vault-url https://particularcodesigning.vault.azure.net `
+          --azure-key-vault-client-id ${{ secrets.AZURE_KEY_VAULT_CLIENT_ID }} `
+          --azure-key-vault-tenant-id ${{ secrets.AZURE_KEY_VAULT_TENANT_ID }} `
+          --azure-key-vault-client-secret ${{ secrets.AZURE_KEY_VAULT_CLIENT_SECRET }} `
+          --azure-key-vault-certificate ${{ secrets.AZURE_KEY_VAULT_CERTIFICATE_NAME }} `
+          src/binaries/Particular.EndpointThroughputCounter.exe
+      - name: Deploy to S3
+        shell: pwsh
+        run: |
+          echo "Creating Particular.EndpointThroughputCounter.zip archive"
+
+          Compress-Archive -Path .src/binaries/* -DestinationPath ./Particular.EndpointThroughputCounter.${{env.MinVerVersion}}.zip
+
+          echo "Uploading zip file to AWS"
+
+          aws s3 cp ./Particular.EndpointThroughputCounter..${{env.MinVerVersion}}.zip s3://particular.downloads/EndpointThroughputCounter/Particular.EndpointThroughputCounter.${{env.MinVerVersion}}.zip --content-type application/zip --acl public-read
```

Comment:
1. What are the 2 dots before the first MinVerVersion?
2. We'll want the S3 path to not have the version in it. We only want to make the most recent version available, and we want the download link stable so we don't have to change docs.